src/UICrafter.Core/Components/PocButtonNoMistakes.razor

```diff
@@ -0,0 +1,66 @@
+@using MudBlazor
+@using UICrafter.Core.Models
+@using System.Diagnostics
+@using Microsoft.AspNetCore.Components
+@using System.Net.Http
+@using System.Net.Http.Json
+@using UICrafter.Core.Utility
+
+@inject HttpClient Http
```

Comment:
Generally you need to use IHttpClientFactory as you could end up getting socket exhaustion. I am uncertain if Blazor hybrid has the exact same issue as ASP.NET Core. But a asking the chat it said that it has the same issues and you should use IHttpClinetFactory in blazor hybrid as well.